Makefile

```diff
@@ -18,6 +18,8 @@ all: ext src bin
 prefix?=/usr
 DESTDIR?=
 VKDTDIR?=$(DESTDIR)$(prefix)/lib/vkdt
+VKDTLIBDIR=/usr/local/lib/vkdt
```

Comment:
probably better to stick to $(DESTDIR)$(prefix) so packagers can drop their temp dirs and put `/usr/local/` in $(prefix) if they want.

Reply:
Agree, but /usr/local contains lib, include, etc., not usr.

Reply:
DESTDIR is only to create packages i think. so to install to /usr/local, you'd set `prefix=/usr/local` and leave DESTDIR empty.